src/ralph/data_importer/widgets.py

```diff
@@ -217,3 +217,29 @@ def render(self, value):
             value.service.name,
             value.environment.name
         )
+
+
+class RackWidget(widgets.ForeignKeyWidget):
+    """Widget for Rack Foreign Key field.
+
+    CSV field format Rack.name
+    """
+    def clean(self, value):
+        if not value:
+            return None
+        else:
+            try:
+                racks = Rack.objects.all()
```

Comment:
Clean method is run for each records in CSV file. 
You can do this more performance optimal.


Pleas add some test for:
1. Import from old ralph (which Rack ID and __str__)
2. Numbers of SQL queries. (assertNumQueries)